app/src/main/java/org/eyeseetea/malariacare/layout/adapters/dashboard/ReviewScreenAdapter.java

```diff
@@ -31,6 +36,23 @@ public ReviewScreenAdapter(List<Value> items, LayoutInflater inflater, Context c
         this.headerLayout = R.layout.review_header;
         this.subHeaderLayout = R.layout.review_sub_header;
         this.recordLayout = R.layout.review_item_row;
+        List<String> colorsList = new ArrayList<>();
+        for (Value value : items) {
+            if (value.getOption() != null && value.getOption().getBackground_colour() != null) {
+                String color = "#" + value.getOption().getBackground_colour();
+                if (!colorsList.contains(color)) {
+                    colorsList.add(color);
+                }
+            }
+        }
+        //Hardcoded colors for a colorList without colors.
+        if (colorsList.size() == 0) {
+            colorsList.add("#4d3a4b");
```

Comment:
@idelcano if this solution is only for cambodia better extract this code in a cambodia strategy